source/includes/stackpath/_delivery_rules.md

```diff
@@ -161,3 +179,32 @@ Attributes | &nbsp;
 ------- | -----------
 `taskId` <br/>*string* | The task id related to the delivery rule deletion.
 `taskStatus` <br/>*string* | The status of the operation.
+
+
+##### Condition
```

Comment:
condition and action are at the bottom of the delete rule docs. would it be more user friendly if they were present  near the create rule docs ? I am saying this because as an external developer, I see ACTIONS and CONDITIONS in the payload but I dont know what they are until I scroll down after the delete docs to see their fields deinition.

Reply:
@razine-bensari that is a very good point. Initially there were next to the create, but since I reused them for the update, there is not point in duplicating them. There is a link to these. 
![image](https://user-images.githubusercontent.com/17802570/114182963-43727380-9911-11eb-8167-4df868820c1e.png)

From here, I can either:
1. leave them at the end
2. Duplicate them for the create and update
3. Put them right after the update

WDYGT?

Reply:
I think since the links are there im fine with it being at the bottom.

Reply:
oh my apologies, did not see the links!!! with links it is good enough, having them at the bottom is fine

Reply:
man i am off today, need to "caffeinate" myself